docs/sources/datasources/elasticsearch.md

```diff
@@ -215,6 +215,8 @@ Once the result is returned, the log panel shows a list of log rows and a bar ch
 
 Note that the fields used for log message and level is based on an [optional data source configuration](#logs).
 
+Log volume panel above the log panel shows accumulated log distribution over time for all log queries. Results are split by level when level field is configured.
+
```

Comment:
The volume panel is shown only in Explore ATM, this is kind of confusing since it seems there's always a volume panel on top of the logs panel.

also, given this is an explore specific behaviour, maybe we'd be better off describing it in `docs/sources/explore/logs-integration.md` ?

wdyt?

Reply:
Good point! I'll move docs to https://grafana.com/docs/grafana/latest/explore/logs-integration/

Reply:
Changed in 1492ff550e660aa6b199d83154efd22c7f1ba2bf